api/candlepin-api-spec.yaml

```diff
@@ -557,7 +557,16 @@ paths:
       tags:
         - cloud_registration
       operationId: cloudAuthorize
```

Comment:
I don't know if openapi-generator allows it, but is it possible to add both the content types in the 200 response here to show the 2 different types of response? And maybe even update the `description` node to explain that when using autoreg 1 you get text/plain, and for autoreg 2 you get application/json. So then the `responses:` node would look like this:
```
      responses:
        200:
          description: Cloud authorization successful. When the version of authorization requested is 2, application/json is returned that contains a JWT token amongst other data. Otherwise, only a JWT token in text/plain format is returned.
          content:
            text/plain:
              schema:
                type: string
            application/json:
              schema:
                $ref: '#/components/schemas/CloudAuthenticationResultDTO'
```

Reply:
It appears that this is possible! Updated